packages/fxa-content-server/app/scripts/lib/oauth-errors.js

```diff
@@ -97,6 +97,14 @@ var ERRORS = {
     errno: 121,
     message: 'Invalid grant_type'
   },
+  SERVER_UNAVAILABLE: {
+    errno: 201,
+    message: t('System unavailable, try again soon')
+  },
+  DISABLED_CLIENT_ID: {
+    errno: 202,
+    message: t('Client has been temporarily disabled.')
```

Comment:
This is a new string, which I guess it not a good idea in a point-release. Perhaps we can have it say "System unavailable, try again soon" for now?

Reply:
> Perhaps we can have it say "System unavailable, try again soon" for now?

I'm good with that and follow up with better message.